File: src/main/java/dev/slne/surf/lobby/jar/command/subcommand/ParkourStartCommand.java
package dev.slne.surf.lobby.jar.command.subcommand;

import dev.jorel.commandapi.CommandAPICommand;

import dev.slne.surf.lobby.jar.PluginInstance;

public class ParkourStartCommand extends CommandAPICommand {

  public ParkourStartCommand(String commandName) {
    super(commandName);

    executesPlayer((player, args) -> {
      PluginInstance.instance().jumpAndRunProvider().start(player);
    });
  }
}

File: src/main/java/dev/slne/surf/lobby/jar/command/subcommand/ParkourStartCommand.kt
package dev.slne.surf.lobby.jar.command.subcommand

import dev.jorel.commandapi.CommandAPICommand
import dev.jorel.commandapi.executors.CommandArguments
import dev.jorel.commandapi.executors.PlayerCommandExecutor
import dev.slne.surf.lobby.jar.JumpAndRunService
import dev.slne.surf.lobby.jar.PluginInstance
import org.bukkit.entity.Player

class ParkourStartCommand(commandName: String) : CommandAPICommand(commandName) {
    init {
        withPermission("jumpandrun.command.start")

        executesPlayer(PlayerCommandExecutor { player: Player, _: CommandArguments? ->
            JumpAndRunService.start(player)
        })
    }
}
